Update release-6.3.0.md (#10631)
pingcap/docs: releases/release-6.3.0.md

* Log backup supports GCS and Azure Blob Storage as backup storage.
* Log backup is now compatible with the `exchange partition` DDL.
* The SQL statement `ALTER TABLE ...SET TiFLASH MODE ...` previously used for enabling [fastscan](/develop/dev-guide-use-fastscan.md) is deprecated, and replaced by the system variable [`tiflash_fastscan`](/system-variables.md#tiflash_fastscan-new-in-v630). When you upgrade from v6.2.0 to v6.3.0, all FastScan settings in v6.2.0 will become invalid, but will not affect the normal reading of data. In this case, you need to configure the variable [`tiflash_fastscan`](/system-variables.md#tiflash_fastscan-new-in-v630) to enable or disable FastScan. When you upgrade from an earlier version to v6.3.0, the FastScan feature is not enabled by default for all sessions to keep data consistent.

> * To deploy TiFlash under Linux with AMD64 architecture, the CPU must support AVX2 instruction sets, use `cat /proc/cpuinfo | grep avx2` to confirm there is output. TiFlash's vectorization engine needs CPU instruction sets to perform a better effect.

Reviewer: `TiFlash's vectorization engine needs CPU instruction sets to perform a better effect.` Can's see any relationship between this sentence and the other two.
```suggestion
* To deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. TiFlash's vectorization engine needs CPU instruction sets to perform a better effect.
```

> * To deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. TiFlash's vectorization engine needs CPU instruction sets to perform a better effect.

Author: ```suggestion
* To deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. By using such CPU instruction sets, TiFlash's vectorization engine can deliver better performance.
```
